contracts/staking/src/execute.rs

```diff
@@ -162,12 +162,12 @@ pub fn execute_liquid_unstake(
 
     STATE.load(deps.storage)?;
 
-    // TODO: lets discuss, added minimum_liquid_stake_amount as a placeholder
+    // TODO: lets discuss, added minimum_liquid_unstake_amount as a placeholder
     // Do we want to add a minimum unstake amount? As time goes on the stake and unstake amounts will diverge
     ensure!(
-        amount >= config.minimum_liquid_stake_amount,
-        ContractError::MinimumLiquidStakeAmount {
-            minimum_stake_amount: (config.minimum_liquid_stake_amount),
```

Comment:
let's instead remove this for now. there is no apparent use case and we can always readd via contract migration

Reply:
Want to double check- Do you mean minimum stake amount? not minimum **unstake** amount?

Reply:
What if we initially set **minimum_stake_amount** = 0 ? Wouldn’t that be an easier way to change in the future?

Reply:
It is for now just dead code. let's add the according code when we actually need it.

Reply:
Removed it :) including the test code

Reply:
Removed it :) including the test code